R/with_parameters.R

```diff
@@ -156,11 +156,22 @@ build_label <- function(..., case_names) {
 }
 
 build_and_run_test <- function(..., .test_name, desc, code, env) {
-  completed_desc <- paste(desc, .test_name)
   args <- list(..., .test_name = .test_name)
+  completed_desc <- glue_data(args, desc)
```

Comment:
Could we think about the ordering for this? We might want to put glue formatting in the test_name too.

*  Concat the test name to the desc provided (following the documentation above)
*  Then populate glue placeholders for that completed string

Reply:
Do you have a use case in mind for that? I tried to accommodate two approaches:

 1. The `desc=` of `with_parameters_test_that()` is glue-formatted, e.g. `with_parameters_test_that("{fmt}", ...)`
 2. The `.test_name` is glue-formatted, e.g. `with_parameters_test_that("...", code, .test_name = "{fmt}")`

Supporting glue-formatted strings in _both_ arguments seemed unnecessary to me.

Reply:
Good point. Let's leave the formatting in the desc.

Thanks!